helpdesk-service/helpdesk-service-client/pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>migbase64</artifactId>
             <version>2.2</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>2.0.2-beta</version>
```

Comment:
Please change it to 1.9.5 version and choose the test scope.

Reply:
ok